core/templates/core/tummytime_form.html

```diff
@@ -30,13 +30,13 @@
     <script type="text/javascript">
         $(function () {
             $('#datetimepicker_start').datetimepicker({
-                format: 'YYYY-MM-DD HH:mm:ss',
+                format: '{{ MOMENTJS_FORMAT }}',
```

Comment:
For this one model (TummyTime), seconds are vital. How can we capture that while also supporting configurable time formats?